openmetadata-ui/src/main/resources/ui/src/styles/x-master.css

```diff
@@ -1323,8 +1323,13 @@ div.ant-typography-ellipsis-custom {
 }
 
 /* Feed text pre tag CSS */
+.activity-feed-card-text {
+  padding-top: 6px;
+}
+
 .activity-feed-card-text pre {
   white-space: pre-wrap; /* Since CSS 2.1 */
+  padding-top: 10px;
```

Comment:
Why we are adding padding-top to `pre` tag?

Reply:
added for testing styles, will remove it.